src/net.cpp

```diff
@@ -1830,7 +1830,8 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
         // Only connect out to one peer per network group (/16 for IPv4).
         int nOutboundFullRelay = 0;
         int nOutboundBlockRelay = 0;
-        std::set<std::vector<unsigned char> > setConnected;
+        std::set<std::vector<unsigned char> > set_connected_full_relay;
+        std::set<std::vector<unsigned char> > set_connected_block_relay;
```

Comment:
nit: I don't believe the `> >` construction is necessary anymore here. See other uses`<*>>` in the codebase.